config/routes.rb

```diff
@@ -1,5 +1,10 @@
 Rails.application.routes.draw do
-  root "projects#index"
+  resources :projects do
+    resources :tasks
+    delete '/task_delete', to: 'tasks#destroy'
+    get '/task_show', to: 'tasks#show'
```

Comment:
Are you really need this?

Reply:
Nope, no actual need for them. I just added them cuz as far as I understood the task, y'all wanna check us for creating the routing for some defs ourselves, so decided to add them, and not rely on the routing provided by the resources for these two functions show & destroy.

Reply:
No no, use `resource` are the best solution!